### PR TITLE
picopass: Add wiegand parsing plugin

### PR DESCRIPTION
**Pull request by bettse**
# What's new

- Add plugin to attempt to parse wiegand data

# Verification 

### Method 1, read card

- read card with H10301 (26 bit) wiegand
- go to more
- see "parse" and select
- See CN and FC

### Method 1, saved card

- select saved card with H10301 (26 bit) wiegand
- go to info
- see "parse" and select
- See CN and FC

# Checklist (For Reviewer)

- [x] PR has description of feature/bug or link to Confluence/Jira task
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,6 @@
 [submodule "air_arkanoid/engine"]
 	path = air_arkanoid/engine
 	url = https://github.com/flipperdevices/flipperzero-game-engine.git
+[submodule "picopass/plugin"]
+	path = picopass/plugin
+	url = https://gitlab.com/bettse/flipper-wiegand-plugin.git
```
Comment (skotopes) [inline .gitmodules:6]: Do you really need separate repository for that?

File: picopass/picopass.c [hunk 2]
```diff
@@ -97,6 +97,27 @@ Picopass* picopass_alloc() {
     view_dispatcher_add_view(
         picopass->view_dispatcher, PicopassViewLoclass, loclass_get_view(picopass->loclass));
 
+    picopass->plugin_manager =
```
Comment (skotopes) [inline picopass/picopass.c:100]: I guess you are planning to add more plugins in future?

Comment (bettse) [reply picopass/picopass.c:100]: No other plugin ideas right now, but I liked the folder form of handling this vs the single plugin form (based on the example apps)

File: .gitmodules (hunk 1 above)
Comment (bettse) [reply .gitmodules:6]: Sharing with Picopass, seader, and can share with Wiegand app (https://github.com/jamisonderek/flipper-zero-tutorials/tree/main/gpio/wiegand) and could be used by RFID app 😉

**Review by skotopes — APPROVED** [review]

**Pull request merged**